packages/web/next.config.js

```diff
@@ -25,6 +25,11 @@ module.exports = withTM(
           destination: '/profile/setup/username',
           permanent: false,
         },
+        {
+          source: '/profile/setup/username',
```

Comment:
may wanna replace all occurences in the codebase of this url with the other one, and also change the redirect just above this to the new one.

Reply:
good catch, thank you. also i'll leave
```

        {
          source: '/profile/setup/username',
          destination: '/profile/setup/name',
          permanent: false,
        },
```
in case someone accesses an old link. afaik i have already changed all the other occurances of `/setup/username`